src/AppKit/NSOpenGLPixelFormat.cs

```diff
@@ -58,7 +58,11 @@ public NSOpenGLPixelFormat (NSOpenGLPixelFormatAttribute[] attribs) : base (NSOb
 		{
 			var list = new List<NSOpenGLPixelFormatAttribute> ();
 			for (int i = 0; i < args.Length; i++){
-				var v = (NSOpenGLPixelFormatAttribute) args [i];
+				NSOpenGLPixelFormatAttribute v;
+				if (args[i] is NSOpenGLPixelFormatAttribute)
+					v = (NSOpenGLPixelFormatAttribute) args [i];
+				else
+					v = ((NSOpenGLPixelFormatAttribute) (int)args [i]);
```

Comment:
I don't think this will result in the behavior the user expects.

According to the bug the user wants to pass `NSOpenGLProfile.Version3_2Core` followed by an integer. This method does not handle `NSOpenGLProfile.Version3_2Core` at all, so it won't be added to the attribute list (neither will the method consume the subsequent integer). At the next iteration, the integer will be cast to an NSOpenGLPixelFormatAttribute, and since there's no enum field whose value is 0, the integer will be ignored as well.

IMHO the following should be done:

* Check if the input value is `NSOpenGLProfile` or `NSOpenGLPixelFormatAttribute`, and handle `NSOpenGLProfile` in a separate switch statement (so that the code will properly process any subsequent integers). Reject anything else with a descriptive exception message ("values must be `NSOpenGLProfile` or `NSOpenGLPixelFormatAttribute`).
* Not silently ignore values we don't understand (my preference would be to throw an exception, but I realize that would be a breaking change by now, so maybe a Console.WriteLine instead?).
* Add tests :smile: